lib/cube_browser/__init__.py

```diff
@@ -539,26 +539,33 @@ def __init__(self, plots):
 
         self._slider_by_name = {}
         self._name_by_slider_id = {}
+        if self._axis_by_name:
+            name_size = max([len(name) for name in self._axis_by_name])
```

Comment:
Basically, it's a character count of the `standard_name` or `long_name` (or `alias`) ... so "not so huge".

Happy to change the variable name.